hppc/src/main/templates/com/carrotsearch/hppc/KTypeVTypeHashMap.java

```diff
@@ -678,6 +679,18 @@ public void ensureCapacity(int expectedElements) {
     }
   }
 
+  @Override
+  public long ramUsageBytes() {
+    return RamUsageEstimator.NUM_BYTES_OBJECT_HEADER + 4 * Integer.BYTES + Double.BYTES + 1 +
```

Comment:
Add comments for magic constants, maybe (so that we know which fields they account for)?

Reply:
Addressed in new commit :)